src/shared/bg_public.h

```diff
@@ -1125,12 +1125,24 @@ enum meansOfDeath_t
 };
 
 // TODO: move other button definitions into gamelogic
```

Comment:
You did the TODO so delete it :)

Reply:
Technically, there are still some though?

Reply:
You got all the ones that aren't used by the engine so there's nothing left that can be easily done.